src/box/lua/feedback_daemon.lua

```diff
@@ -26,13 +26,157 @@ local function get_fiber_id(f)
     return fid
 end
 
-local function fill_in_feedback(feedback)
+local function detect_docker_environment_impl()
+    local fh = fio.open('/proc/1/cgroup', {'O_RDONLY'})
+    if not fh then
+        return false
+    end
+
+    -- fh:read() doesn't read empty "proc" files
+    local big_enough_chunk = 4096
+    local s = fh:read(big_enough_chunk)
+    fh:close()
+
+    return s and s:find('docker') and true or false
+end
+
+local cached_detect_docker_env
+
+local function detect_docker_environment()
+    if cached_detect_docker_env == nil then
+        cached_detect_docker_env = detect_docker_environment_impl()
+    end
+
+    return cached_detect_docker_env
+end
+
+local function is_system_space(sp)
+    return box.schema.SYSTEM_ID_MIN <= sp.id and
+            sp.id <= box.schema.SYSTEM_ID_MAX
+end
+
+local function is_jsonpath_index(idx)
+    for _, part in pairs(idx.parts) do
+        if part.path ~= nil then
```

Comment:
Such approach doesn't distinguish jsonpath indexes from multikey indexes. It's a bit different things.

Reply:
Done. Do not increment a counter if `path` contains `[*]` substring

Reply:
I suppose you should add separate metric for multikey indexes. E.g. why are functional indexes better than multikey (you count functional indexes but skip multikey)?

Reply:
I agree. There is also functional multikey index, which is also worth count

Reply:
Multikey indices should be considered as jsonpath indices, IMHO.

Reply:
@olegrok Added multikey metric for both jsonpath and functional indices.
@Totktonada Include multikey indices in jsonpath counter. Same for functional multikey indices.